tests/data/fet_private_key.txt

```diff
@@ -0,0 +1 @@
+ccf59158ee8c5bc1aa5422f511f64d7ed55106a38806259568d39b158a1719e7
```

Comment:
Is this correct that we are committing a private key (presumably then everyone running this will be using the same wallet)?